spec/acceptance/firewall_spec.rb

```diff
@@ -833,6 +833,33 @@ class { '::firewall': }
     end
   end
 
+  describe 'checksumfill' do
+    context 'virbr' do
+      it 'applies' do
+        pp = <<-EOS
+          class { '::firewall': }
+          firewall { '576 - test':
+            proto  => udp,
+            table  => 'mangle',
+            outiface => 'virbr0',
+            chain  => 'POSTROUTING',
+            dport => '68',
+            jump  => 'CHECKSUM',
+            checksumfill => true,
+          }
+        EOS
+
+        apply_manifest(pp, :catch_failures => true)
+      end
+
+      it 'should contain the rule' do
+        shell('iptables-save -t mangle') do |r|
+          expect(r.stdout).to match(/-A POSTROUTING -o virbr0 -p udp -m multiport --dports 68 -m comment --comment "576 - test" -j CHECKSUM --checksumfill/)
```

Comment:
It looks like the actual option is `--checksum-fill`, so I think this is going to fail.